.gitignore

```diff
@@ -1,2 +1,2 @@
 AutoscoperM/__pycache__/*
-
+build*/
```

Comment:
It is recommended to do out-of-source build

Reply:
This is relevant:
* https://github.com/BrownBiomechanics/SlicerAutoscoperM/pull/22